Makefile

```diff
@@ -44,6 +44,7 @@ vet:
 # Generate code
 generate:
 	go generate ./pkg/... ./cmd/...
+	/usr/local/bin/bash  ./hack/update-codegen.sh
```

Comment:
Can you rely on the shebang in `./hack/update-codegen.sh` instead? Most people won't have `/usr/local/bin/bash`.

Reply:
Thanks for this reminder. I fixed it to work with linux and mac.